playground/mocks/config/responseConfig.js

```diff
@@ -102,6 +102,9 @@ const idx = {
     // 'authenticator-verification-password',
     // 'authenticator-verification-okta-verify-totp',
     // 'authenticator-verification-okta-verify-push',
+    // 'authenticator-verification-google-authenticator',
+    // 'factor-verification-password',
+    // 'factor-verification-email',
```

Comment:
nit: remove the factor ones we have removed all the old code so dont need it anymore